sdk/src/main/scala/io/horizen/fork/ActiveSlotCoefficientFork.scala

```diff
@@ -0,0 +1,13 @@
+package io.horizen.fork
+
+case class ActiveSlotCoefficientFork(
+                                    activeSlotCoefficient: Double = -1
+                                    ) extends OptionalSidechainFork
```

Comment:
let's add a check that  activeSlotCoefficient (if defined) is > 0

Reply:
Done